ibm_db2/datadog_checks/ibm_db2/ibm_db2.py

```diff
@@ -65,6 +75,21 @@ def check(self, instance):
         self.query_table_space()
         self.query_transaction_log()
         self.query_custom()
+        self.collect_metadata()
+
+    def collect_metadata(self):
+        try:
+            version = get_version(self._conn)
```

Comment:
According to https://github.com/ibmdb/python-ibmdb/wiki/APIs#ibm_dbserver_info
The `ibm_db.get_db_info` method never raises an exception but instead return False.
I'd say let's keep the try catch block (in case the doc is not accurate/all versions do not behave the same). But let's make sure that we handle the case where "version" is "False"

Reply:
sure! Added the try/except to catch the `connection closed` issue since the function uses the connection object.